tst/LargeAlgebras.tst

```diff
@@ -0,0 +1,83 @@
+gap> START_TEST( "LargeAlgebras.tst");
+
+#
```

Comment:
Some more suggestions: maybe a short descriptions in these comment lines with a brief characterisation of the example?